.travis.yml

```diff
@@ -4,6 +4,8 @@ branches:
 
 language: cpp
 
+sudo: required
```

Comment:
I added that because Travis said: the job is running on container-based infrastructure, which does not allow use of 'sudo', setuid and setguid executables.

This was the solution to getting nupic on my local machine to build on travis.

https://travis-ci.org/akhilaananthram/nupic/jobs/66064665